server/server.js

```diff
@@ -2,18 +2,30 @@ const express = require('express')
 const ws = require('ws')
 const cors = require('cors')
 const server = express()
-
+let map = new map(); //creating map for uuid as key and token as value
```

Comment:
i'm afraid that here will be an error. Because JS has `Map` class but hasn't  `map`

Reply:
it was fixed